### DISCUSSION TITLE
OctoPrint Plugin development

### DISCUSSION THREAD

**Discussion by mkevenaar** [Donkie/Spoolman#85] [General]
Hi @Donkie 

I want to let you know I am working on a companion [plugin for OctoPrint](https://github.com/mkevenaar/OctoPrint-Spoolman).

At first, it will only report usage back, I do not intend to rewrite the UI. 

I'll keep you posted.

**Comment by Donkie** [comment]
Hey, that's sick! Please do :)

**Comment by mkevenaar** [comment]
I would like to request to use the Spoolman logo

**Reply by Donkie** [reply]
Yes, that's fine :)
